### PR TITLE
Include learn resources to online courses

### LINKED ISSUE #200: Link teaching resources and course material?
Is it on scope for the webpage to link external course material or teaching resources from the *learn* page? I'm mainly thinking about pure Fortran programming courses and also more domain-specific specific courses that include Fortran introductions to teach other more specific concepts.

If we agree that it is on scope, what would be criteria to list a course on the *learn* page?

### PR DESCRIPTION
**Pull request by awvwgk**
Just the awesome list collected by @ivan-pi in https://github.com/fortran-lang/fortran-lang.org/issues/200#issuecomment-776134553 for now.

Closes #200

### CODE REVIEW

**Review by LKedward — APPROVED** [review]
This is great! Thanks @awvwgk and @ivan-pi.

**Comment by LKedward** [comment]
#build_preview

File: _data/learning.yml [hunk 1]
```diff
@@ -89,6 +89,130 @@ reference-links:
...
+    url: https://web.stanford.edu/class/me200c/tutorial_90/
+    description: >-
+      older tutorial by Paul Hargrove and Sarah Whitlock, Stanford University
+
+  - name: "Fortran 90 Tutorial"
+    url: https://pages.mtu.edu/%7eshene/COURSES/cs201/NOTES/fortran.html
+    description: >-
+      by C.-K. Shene, Michigan Technologial University
+
+
```
Comment (milancurcic) [inline _data/learning.yml:215]:
```suggestion
  - name: "Exploring Modern Fortran Basics"
    url: https://www.manning.com/books/exploring-modern-fortran-basics
    description: >-
      by Milan Curcic, excerpt (Chapters 2, 3, and 4) from Modern Fortran - Building Efficient Parallel Applications

```

**Review by milancurcic — APPROVED** [review]

**Comment by milancurcic** [comment]
Thank you, this has been open for a while without complaints, so I will merge.

**Comment by awvwgk** [comment]
#delete_preview

**Pull request merged**